Tests/LibCrypto/TestRSA.cpp

```diff
@@ -25,8 +25,9 @@ TEST_CASE(test_RSA_raw_encrypt)
         "8126832723025844890518845777858816391166654950553329127845898924164623511718747856014227624997335860970996746552094406240834082304784428582653994490504519"_bigint,
         "65537"_bigint,
     });
-    u8 buffer[rsa.output_size()];
-    auto buf = Bytes { buffer, sizeof(buffer) };
+    Vector<u8> buffer = {};
```

Comment:
Couldn't all these `Vector<u8>`s be simply `ByteBuffer`?

Reply:
Didn't know we had those. Would ByteBuffer be better? Also does it have a resize?

Reply:
`ByteBuffer` is basically made for handling bytes, and is easily convertible into `Bytes` or `ReadonlyBytes`. It does have a `::resize()`, check out `AK/ByteBuffer.h` for yourself!

Reply:
Ok I'll move to that and rebase.